lib/service_base_url_test_kit/service_base_url_validate_group.rb

```diff
@@ -172,17 +195,39 @@ def get_endpoint_availability_limit(endpoint_availability_limit)
           .uniq
 
         check_limit = get_endpoint_availability_limit(endpoint_availability_limit)
+        one_endpoint_valid = false
 
         endpoint_list.each_with_index do |address, index|
           assert_valid_http_uri(address)
 
-          next if check_limit.present? && index >= check_limit
+          next if endpoint_availability_success_rate == 'none' || (check_limit.present? && index >= check_limit)
 
           address = address.delete_suffix('/')
           get("#{address}/metadata", client: nil, headers: { Accept: 'application/fhir+json' })
```

Comment:
if this results in a DNS issue or TLS issue I think this will cause an exception to be raised.  It feels a bit redundant, but i think you will need to put this in twice below so this can be in the warning block which I think catches those kinds of errors.

Reply:
Turns out this works (move that get down into the warning block and a dns issue will fix it).  However, a little lower down there will be a problem with response[:status] so you'll have to change that line to:

`if !one_endpoint_valid && response && response[:status] == 200 && resource.present? &&`

(note adding that check that there is even a response object)

Reply:
Oh, I checked this by grabbing the bundle from the url, pasting that into the bundle input, and then updating the endpoint's domain to be something like 'http://localhoastestasdfsdafsdaf' which causes an exception to be raised (and then rescued if you put it in the warning block).